dist/lib/test.js

```diff
@@ -94,9 +94,9 @@ function testGuetzliEncode(t) {
     // 1. load png
     var m1 = helper.loadImage(testdir + '/bees.png');
     // 2. guetzli encode
-    var jepgData = guetzli.encodeImage(m1);
+    var jpegData = guetzli.encodeImage(m1);
     // 3. decode jpeg
-    var m2 = helper.decodeJpg(jepgData);
```

Comment:
the `dist/*` is the output files, you should fix the [lib/test.ts](https://github.com/chai2010/guetzli-js/blob/master/lib/test.ts#L73), then run `gulp dist` to update the output files.